### PR TITLE
Use env variables instead of hardcoding key for Honeycomb plugin example

### PR DESCRIPTION
**Pull request by phpb-com**
Update Honeycomb example to use environment variables to store API key

### CODE REVIEW

File: content/pages/platform/functions/plugins/honeycomb.md [hunk 1]
```diff
@@ -20,12 +20,17 @@ $ npm install @cloudflare/pages-plugin-honeycomb
...
+import honeycombPlugin from '@cloudflare/pages-plugin-honeycomb'
+
+export const onRequest: PagesFunction<{
+  HONEYCOMB_API_KEY: string
+  HONEYCOMB_DATASET: string
+}> = async context => {
+  return honeycombPlugin({
+    apiKey: context.env.HONEYCOMB_API_KEY,
+    dataset: context.env.HONEYCOMB_DATASET,
+  })(context)
```
Comment (GregBrimble) [inline content/pages/platform/functions/plugins/honeycomb.md]:
```suggestion
}> = (context) => {
  return honeycombPlugin({
    apiKey: context.env.HONEYCOMB_API_KEY,
    dataset: context.env.HONEYCOMB_DATASET,
  })(context);
```

**Comment by GregBrimble** [comment]
Not sure if the dataset name is private in the same way that an API key might be, but this is probably a good PR, thank you!

We show both ways on [the Sentry page](https://github.com/cloudflare/cloudflare-docs/blob/production/content/pages/platform/functions/plugins/sentry.md), which I think is quite good:

image

Worth adding here too? If not, at least a sentence about saying how these are now being loaded from environment variables would give the reader the context they need to know how to set these values.

**Comment by phpb-com** [comment]
Thanks for looking into this and suggested edits. Let me update the PR.

**Review by deadlypants1973 — APPROVED** [review]

**Comment by phpb-com** [comment]
Ok, I updated the usage example to clarify usage of environment variables, and added back the hardcoded example.

File: content/pages/platform/functions/plugins/honeycomb.md [hunk 2]
```diff
@@ -16,12 +16,32 @@ $ npm install @cloudflare/pages-plugin-honeycomb
 
 ## Usage
 
+The following usage example uses environment variables you will need to set in your Pages settings.
```
Comment (GregBrimble) [inline content/pages/platform/functions/plugins/honeycomb.md]:
```suggestion
The following usage example uses environment variables you will need to set in your Pages project settings. 
```

File: content/pages/platform/functions/plugins/honeycomb.md [hunk 3]
```diff
@@ -16,12 +16,32 @@ $ npm install @cloudflare/pages-plugin-honeycomb
...
 ## Usage
 
+The following usage example uses environment variables you will need to set in your Pages settings. 
+
 ```typescript
 ---
 filename: functions/_middleware.ts
 ---
-import honeycombPlugin from "@cloudflare/pages-plugin-honeycomb";
+import honeycombPlugin from '@cloudflare/pages-plugin-honeycomb'
```
Comment (GregBrimble) [inline content/pages/platform/functions/plugins/honeycomb.md]:
```suggestion
import honeycombPlugin from "@cloudflare/pages-plugin-honeycomb";
```

File: content/pages/platform/functions/plugins/honeycomb.md [hunk 4]
```diff
@@ -16,12 +16,32 @@ $ npm install @cloudflare/pages-plugin-honeycomb
...
+}
+```
 
+Alternatively, you can hardcode (not advisable for API key) your settings the following way:
+
+```typescript
+---
+filename: functions/_middleware.ts
+---
+import honeycombPlugin from "@cloudflare/pages-plugin-honeycomb";
```
Comment (GregBrimble) [inline content/pages/platform/functions/plugins/honeycomb.md]:
```suggestion
import honeycombPlugin from "@cloudflare/pages-plugin-honeycomb";

```

File: content/pages/platform/functions/plugins/honeycomb.md [hunk 5]
```diff
@@ -16,12 +16,32 @@ $ npm install @cloudflare/pages-plugin-honeycomb
...
+  HONEYCOMB_DATASET: string
+}> = (context) => {
+  return honeycombPlugin({
+    apiKey: context.env.HONEYCOMB_API_KEY,
+    dataset: context.env.HONEYCOMB_DATASET,
+  })(context);
+}
+```
 
+Alternatively, you can hardcode (not advisable for API key) your settings the following way:
```
Comment (GregBrimble) [inline content/pages/platform/functions/plugins/honeycomb.md]:
```suggestion
Alternatively, you can hard-code (not advisable for API key) your settings the following way:
```

**Review by GregBrimble — APPROVED** [review]
Couple of nits, but yeah, that's perfect now, thank you!

File: content/pages/platform/functions/plugins/honeycomb.md (hunk 2 above)
Comment (phpb-com) [reply content/pages/platform/functions/plugins/honeycomb.md]: Funny part is, I wanted to add project settings, but after looking at the interface, decided against it. But yes, that does sound better.

**Pull request merged**